packages/ruby/spec/readme/metrics_spec.rb

```diff
@@ -1,5 +1,45 @@
+require "rack/test"
+require "webmock/rspec"
+
 RSpec.describe Readme::Metrics do
+  include Rack::Test::Methods
+
+  before do
+    stub_request(:post, readme_endpoint)
+  end
+
   it "has a version number" do
     expect(Readme::Metrics::VERSION).not_to be nil
   end
+
+  it "doesn't fail" do
```

Comment:
Could this test be deleted since the other test verifies behavior via WebMock?

I'm guessing we'd want to cover the case where the request times out/fails but in separate tests?

Reply:
The WebMock only tests the external request, it doesn't test the return value of the middleware itself. I suppose a better assertion would be that the middleware does not change the response of the wrapped app, although in this case they are both hard-coded to 200.

Reply:
We ended up changing the test so that it checked that the response of an app both with and without middleware is the same.